tools/ffi_coverage/src/allowlist.rs

```diff
@@ -412,7 +414,17 @@ lazy_static::lazy_static! {
 
         // FFI largely deals with primitives rather than Rust's nice wrapper types
         // (which are hard to do in a zero-cost way over FFI)
-        "icu::calendar::types",
```

Comment:
Note: I stopped filtering out the whole `icu::calendar::types` module since the types inside are homogeneous. I went through each one and allowlisted only the ones that are actually wrapper types.